Lombiq.VueJs.Samples/Views/QrCard/Index.cshtml

```diff
@@ -0,0 +1,23 @@
+@model Lombiq.VueJs.Samples.ViewModels.QrCardAppViewModel
+
+<vue-component area="@FeatureIds.Area" name="@ResourceNames.QrCard" />
+<div id="qrCardApp">
+    <qr-card
+        :error-messages="{
+                NotAllowedError: '@T["You need to grant camera access permission."]',
+                NotFoundError: '@T["No camera found."]',
+                NotSupportedError: '@T["Secure context required (HTTPS, localhost)."]',
+                NotReadableError: '@T["The camera already in use."]',
+                OverconstrainedError: '@T["Installed cameras are not suitable."]',
+                StreamApiNotSupportedError: '@T["The stream API is not supported in this browser."]',
+                InsecureContextError: '@T["Camera access is only permitted in secure context. Use HTTPS or localhost rather than HTTP."]',
+                Unknown: '@T["Camera error."]',
+        }"
+        api-url="@Model.ApiUrl"></qr-card>
```

Comment:
What would you say about wrapping the closing tag to a new line for better visibility?
```suggestion
        api-url="@Model.ApiUrl">
    </qr-card>
```

Reply:
In that case, do the same in _[ContentPart.Edit-BusinessCard.cshtml](https://github.com/Lombiq/Orchard-Vue.js/pull/81/files#diff-abc8774add695e5a8f7451b7d9dd57ad6a054599ce9cd2452104cde144cbfb6fR10)_, too, please.